country-a-service/epps-api/epps-minlog-api/src/main/resources/wsdl/fetch-wsdl-sfsk.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion

set -o errexit
set -o nounset
set -o pipefail

```

I usually add this to all my scripts.
https://medium.com/factualopinions/consider-starting-all-your-bash-scripts-with-these-options-74fbec0cbb83

Reply:
And also it's a good idea to run shellcheck on all shell scripts.  It's fairly easy to integrate in an IDE.